Rocket.Chat/Models/User/UnmanagedUser.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UnmanagedUser.swift
+//  Rocket.Chat
+//
+//  Created by Samar Sunkaria on 8/21/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import DifferenceKit
+
+struct UnmanagedUser: UnmanagedObject, Equatable {
```

Comment:
Same thing I said for UnmanagedSubscription

Reply:
Same thing